front/lib/api/data_sources.ts

```diff
@@ -530,18 +538,28 @@ export async function handleDataSourceSearch({
   searchQuery,
   dataSource,
   dataSourceView,
+  auth,
```

Comment:
The use of the flag here is new compared to my earlier commit. This is to exercise EU embedding in actually search scenarios.